src/init/starship.ps1

```diff
@@ -1,38 +1,79 @@
 #!/usr/bin/env pwsh
 
+function Get-Cwd {
```

Comment:
Would you please prefix these with `starship-` or find some way to avoid getting these functions into the shell scope?

Reply:
I've moved these function declarations into the `global:prompt` function itself so they no longer leak into the user's shell scope.